src/llmcompressor/modifiers/quantization/gptq/utils/gptq_wrapper.py

```diff
@@ -272,7 +294,13 @@ def free(self):
         delattr(self, "H")
         super().free()
 
-    def log_metrics(self, start_tick: float, losses: torch.Tensor):
+    def _update_quantization_parameters(self, args: QuantizationArgs, W: torch.Tensor):
+        observer = MemorylessObserver(args)
```

Comment:
I would make a func in class QuantizationArgs

def get_memory_less_observer(): ...

and use it to get the observer. 

Similar to 
https://github.com/neuralmagic/compressed-tensors/blob/ef77dca6c0d9b25e712f59a0d3f3ec8c4a81ce5d/src/compressed_tensors/quantization/quant_args.py#L98

Reply:
The `QuantizationArgs.get_observer` case is useful when the caller doesn't know what kind of observer to get. The function does the job of inferring which type of observer to return.

In this instance, I think it's a lot clearer to directly instantiate a memoryless observer object. Breaking out another function on `QuantizationArgs` only serves to add another layer of abstraction to a reader.

Reply:
This also isn't a great pattern for future cases when we want to get specific observers, since it means we'd need to create a new function for each observer class

Reply:
No its to abstract away the observers in llm_compressor, obtaining the observer from the quantization arg entrypoint

Reply:
LC already manipulates and calls observers, I don't a reason why LC can't instantiate observers?